Test/comp/Class.dfy

```diff
@@ -1,7 +1,8 @@
-// RUN: %dafny /compile:3 /spillTargetCode:2 /compileTarget:cs "%s" > "%t"
```

Comment:
Where does the directory nesting `comp` originate from?

Reply:
I don't understand your question.